src/gui/mzroll/controller.h

```diff
@@ -15,7 +16,7 @@ class Controller:  public QObject
     Controller();
     ~Controller();
     MainWindow* getMainWindow() { return _mw; }
-
+    PollyIntegration* _iPolly;
```

Comment:
Let us not have names of non-private members starting with an underscore.

Reply:
Ah, yes! 
Sorry about this